### PR TITLE
Add a dedicated service account for the console

### PR DESCRIPTION
**Pull request by fantapsody**
Fixes https://github.com/streamnative/charts/issues/687

### Motivation

Add a dedicated service account for the console.

### Modifications

Add a dedicated service account for the console.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: charts/sn-platform/templates/streamnative-console/streamnative-console-service-account.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name: {{ template "pulsar.streamnative_console.serviceAccount" . }}
+  namespace: {{ template "pulsar.namespace" . }}
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+    component: {{ .Values.streamnative_console.component }}
+  annotations:
+{{- with .Values.streamnative_console.serviceAccount.annotations }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+{{- with .Values.streamnative_console.serviceAccount.annotations }}
```
Comment (FushuWang) [inline charts/sn-platform/templates/streamnative-console/streamnative-console-service-account.yaml:33]: This part is repeated.

File: charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml [hunk 2]
```diff
@@ -191,6 +191,11 @@ spec:
           - name: streamnative-console-data
             mountPath: /data
 
+          securityContext:
+            capabilities:
+              add:
+              - SETUID
```
Comment (maxsxu) [inline charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml:197]: Should this block be put in a separate PR?

Comment (maxsxu) [inline charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml]: Should this block be put in a separate PR?

**Review by maxsxu — APPROVED** [review]
LGTM

**Review by FushuWang — APPROVED** [review]
